contrib/rspamd/template.py

```diff
@@ -0,0 +1,62 @@
+pkgname = "rspamd"
+pkgver = "3.7.5"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCONFDIR=/etc/rspamd",
+    "-DENABLE_FASTTEXT=ON",
+    "-DENABLE_HYPERSCAN=OFF",
+    "-DENABLE_LUAJIT=OFF",
+    "-DENABLE_URI_INCLUDE=ON",
+    "-DRSPAMD_GROUP=_rspamd",
+    "-DRSPAMD_USER=_rspamd",
+    "-DSYSTEM_FMT=ON",
+    "-DSYSTEM_XXHASH=ON",
+    "-DSYSTEM_ZSTD=ON",
+    "-D_CAN_RUN=0",
+    "-DHAVE_ATOMIC_BUILTINS_EXITCODE=0",
+]
+make_build_args = ["--target", "all", "check"]
+hostmakedepends = ["cmake", "ninja", "perl", "pkgconf", "ragel"]
+makedepends = [
+    "elfutils-devel",
+    "fasttext-devel",
+    "fmt-devel",
+    "glib-devel",
+    "icu-devel",
+    "libsodium-devel",
+    "libunwind-devel",
+    "lua5.4-devel",
+    "openssl-devel",
+    "pcre2-devel",
+    "snowball-devel",
+    "sqlite-devel",
+    "xxhash-devel",
+    "zstd-devel",
+]
+pkgdesc = "Spam filtering system"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Apache-2.0 AND BSD-1-Clause AND BSD-2-Clause AND BSD-3-Clause AND CC0-1.0 AND LGPL-2.1-or-later AND LGPL-3.0-only AND MIT AND Zlib"
```

Comment:
where do all these come from?

Reply:
the bundled dependencies https://github.com/rspamd/rspamd/blob/master/contrib/DEPENDENCY_INFO.md and the web interface https://github.com/rspamd/rspamd/tree/master/interface.

Looks like BSD-1-Clause and LGPL-2.1-or-later can be removed and boost needs adding